lms/djangoapps/django_comment_client/forum/views.py

```diff
@@ -45,6 +50,29 @@
 log = logging.getLogger("edx.discussions")
 
 
+class DiscussionCourseViewType(CourseViewType):
+    """
+    A tab for the cs_comments_service forums.
+    """
+
+    name = 'discussion'
+    title = _('Discussion')
+    priority = None
+    view_name = 'django_comment_client.forum.views.forum_form_discussion'
+
+    @classmethod
+    def is_enabled(cls, course, user=None):
+        has_course_access = True
+        if user:
+            has_course_access = (has_access(user, 'staff', course) or
```

Comment:
I think this is the same as the behavior provided by ``EnrolledCourseViewType``? We should just subclass from it instead.